dot/network/service.go

```diff
@@ -139,10 +139,10 @@ func NewService(cfg *Config) (*Service, error) {
 	var bufPool *sizedBufferPool
 	if cfg.noPreAllocate {
 		bufPool = &sizedBufferPool{
-			c: make(chan *[maxMessageSize]byte, cfg.MaxPeers*3),
+			c: make(chan *[maxMessageSize]byte, cfg.MinPeers*3),
```

Comment:
question: why *3?

Reply:
there's currently 3 notifications protocols supported, so the number of open streams  is likely to be peercount * 3 (I think the comment explains this)